### PR TITLE
MRG: Use CI-helpers

### PR DESCRIPTION
**Pull request by Eric89GXL**
Updates our CI builds to use CI-helpers. ~~Also tries to use conda-forge to get Mayavi on Py3k.~~ We'll see if it works.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
This one is good to go. The only downside seems to be that the newest `matplotlib` is soul-crushingly slow in `Report` rendering, making one test there take over 4 minutes. The only real ways to fix this would be to speed up our plotting, fix the mpl regression, or skip the test. :(

File: mne/viz/utils.py [hunk 1]
```diff
@@ -1050,11 +1050,11 @@ def _process_times(inst, times, n_peaks=None, few=False):
     if isinstance(times, string_types):
         if times == "peaks":
             if n_peaks is None:
-                n_peaks = 3 if few else 7
+                n_peaks = min(3 if few else 7, len(times))
             times = _find_peaks(inst, n_peaks)
         elif times == "auto":
             if n_peaks is None:
-                n_peaks = 5 if few else 10
+                n_peaks = min(5 if few else 10, len(times))
```
Comment (jaeilepp) [inline mne/viz/utils.py]: You don't need these. times = 'auto' or 'peaks'

Comment (Eric89GXL) [inline mne/viz/utils.py]: Ahh I need to use `inst.times`, thanks

**Comment by Eric89GXL** [comment]
Times bug fixed, ready from my end

**Comment by mmagnuski** [comment]
so green :sparkles: :open_mouth:

**Comment by agramfort** [comment]
thx heaps @Eric89GXL

**Pull request merged**